scripts/PauseAtHeight.py

```diff
@@ -42,6 +42,14 @@ def getSettingDataString(self):
                     "unit": "mm",
                     "type": "float",
                     "default_value": 0
+                },
+                "extrude_ammount":
```

Comment:
Just trying to be consistent with https://github.com/nallath/PostProcessingPlugin/blob/master/scripts/PauseAtHeight.py#L38 ;) No just kidding, I should probably fix both.